atd-etl/requirements.txt

```diff
@@ -7,4 +7,4 @@ web-pdb==1.*
 sodapy==2.*
 boto3==1.*
 mail-parser==3.*
-python-magic==0.*
\ No newline at end of file
+python-magic==0.*
```

Comment:
to be sure, the cr3 download script no longer uses this requirements file, right? do other scripts that use this requirements file need python-magic?

Reply:
The cr3 download script does still use it to validate that files are pdfs before storing them in S3. But, you are right! This requirement should be removed here since it is covered in requirements for the script that is split out here. The only other place it is used in this repo is a toolbox script that has its own requirements.txt. 🙏 I'll remove it here.